packages/realm/src/platform/node/index.ts

```
@@ -18,5 +18,6 @@
 
 import "./fs";
 import "./device-info";
+import "./sync-proxy-config";
```

Should this be done in `react-native` as well?

---

Alternatively, we could just remove the `react-native` implementation and let it throw.